### PR TITLE
a11y: update about page

### LINKED ISSUE #2125: [Accessibility][Name,Role,value - About]: Ensures buttons have discernible text.
[edit by @corinagum] 
The issues filed below have been addressed, but there are a couple other related problems that I would like addressed in this issue. To quote my comment [below](https://github.com/microsoft/BotFramework-Composer/issues/2125#issuecomment-613142023): 

> [R]elated to this PR, I believe the about page needs to be modified to remove `tabIndex={-1}`, and the `<IconBotton>` should be changed to just an icon inside `<Link>`

Once the above is addressed, we can return this to 'waiting for verification'

---

**User Experience:**
This issue should be resolved for good coding practice if buttons have discernible text.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Select 'About' from left navigation.
5. Run K4W tool on the page and Observe the issue

**Actual Result:**
'Terms of Use' and 'Privacy' buttons does not have discernible text. Screen reader read only as button.

This accessibility issue was found using Accessibility Insights for Web(keros), a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at http://aka.ms/Keros.

**Expected Result:**
Ensures buttons i.e. 'Terms of Use' and 'Privacy' have discernible text. Screen reader should read the name of the button.

**Note:**
Same issue observed for 'Design Flow' pane.
Same issue observe throughout the website for every property pane. {Attached some of screenshot below, please look the issue throughout the application}

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Attachment:**
[MAS4.1.2_About_K4W.zip](https://github.com/microsoft/BotFramework-Composer/files/4266576/MAS4.1.2_About_K4W.zip)
ensures button have discernible text

### PR DESCRIPTION
**Pull request by beyackle**
## Description

This updates the links on the About page to be a little more a11y-friendly.

## Task Item

closes #2125

### CODE REVIEW

**Comment by liweitian** [comment]
a similar issue is https://github.com/microsoft/BotFramework-Composer/issues/2091. And I just open a pr for https://github.com/microsoft/BotFramework-Composer/issues/2091. If this pr also solved https://github.com/microsoft/BotFramework-Composer/issues/2091. You can close https://github.com/microsoft/BotFramework-Composer/pull/2661.

**Comment by beyackle** [comment]
#2091 looks to be a completely different issue, related to how Narrator in Scan Mode will read the label of a control as part of the control itself, and then read that label again when it sees its text.

File: Composer/packages/client/src/pages/about/index.tsx [hunk 1]
```diff
@@ -76,32 +76,14 @@ export const About: React.FC<RouteComponentProps> = () => {
...
-              ariaLabel={formatMessage('Terms of User Button')}
-              iconProps={{ iconName: 'BlockedSite' }}
-              tabIndex={-1}
-            />
-            <Link
-              href={'https://github.com/microsoft/BotFramework-Composer/blob/stable/LICENSE.md'}
-              tabIndex={-1}
-              target={'_blank'}
-            >
+            <Icon styles={about.icon} iconName={'BlockedSite'} tabIndex={-1} />
```
Comment (corinagum) [inline Composer/packages/client/src/pages/about/index.tsx:79]: Does `<Link>` allow an icon to be a part of the text? it looks wonky atm: 
image

Comment (beyackle) [reply Composer/packages/client/src/pages/about/index.tsx:79]: Link doesn't, but I agree it looks wonky. I'll fix that up.

**Pull request merged**